src/index.ts

```diff
@@ -12,6 +13,19 @@ if ('SENTRY_DSN' in process.env
   });
 }
 
+if (process.env.TEMPORARY_FILE_S3_BUCKET === undefined) {
+  throw new SystemConfigurationError('TEMPORARY_FILE_S3_BUCKET must be populated in order to upload to s3.');
+}
+if (process.env.AWS_ACCESS_KEY_ID === undefined) {
+  throw new SystemConfigurationError('AWS_ACCESS_KEY_ID must be populated in order to upload to s3.');
+}
+if (process.env.AWS_ACCESS_SECRET === undefined) {
+  throw new SystemConfigurationError('AWS_ACCESS_SECRET must be populated in order to upload to s3.');
+}
+if (process.env.TEMPORARY_FILE_S3_BUCKET_REGION === undefined) {
+  throw new SystemConfigurationError('TEMPORARY_FILE_S3_BUCKET_REGION must be populated in order to upload to s3.');
+}
+
```

Comment:
This is exactly the same check in the `TemporaryFile` class. Can you wrap this check in a function and call it in these two places?

Not a hill I would die on though.